crates/forge_main_neo/src/domain/action.rs

```diff
@@ -8,6 +8,7 @@ use crate::domain::{CancelId, Timer};
 pub enum Action {
     CrossTerm(Event),
     Initialize,
+    NewConversation,
```

Comment:
Spotlight selection will automatically trigger an action. We don't need a special new conversation action.